.circleci/config.yml

```diff
@@ -164,6 +164,53 @@ jobs:
       - run: make -C build coverage
       - run: /bin/bash <(curl -s https://codecov.io/bash) || echo "Codecov did not collect coverage reports"
 
+  clang-build-debug:
+    docker:
+      - image: valhalla/valhalla:build-latest
+    steps:
+      - checkout
+      - run: ./scripts/format.sh && ./scripts/error_on_dirty.sh
+      - run: node ./scripts/validate_changelog.js
+      - run: git submodule sync && git submodule update --init
+      - restore_cache:
+          keys:
+            - ccache-debug-linux-x86_64-{{ .Branch }}
+            - ccache-debug-linux-x86_64
+      - run: npm install --ignore-scripts
+      - run: mkdir build
+      - run: ls -la /usr/bin/
+      - run: clang-5.0 --version
+      - run: |
+          # NOTE: -Werror disabled in CI, as we currently have >4k warnings.
+          cd build \
+            && cmake .. -DCMAKE_BUILD_TYPE=Debug -DENABLE_SANITIZER=ON -DBUILD_SHARED_LIBS=Off -DENABLE_NODE_BINDINGS=Off -DCMAKE_C_COMPILER=clang-5.0 -DCMAKE_CXX_COMPILER=clang++-5.0
+      - run: export ASAN_OPTIONS=detect_leaks=0 && make -C build -j2 tests check
+      # Note: we save the cache here before doing linting so that if linting fails, we can rebuild quickly
+      # for follow-up fixes
+
+  sanitizer-build-debug:
+    docker:
+      - image: valhalla/valhalla:build-latest
+    steps:
+      - checkout
+      - run: ./scripts/format.sh && ./scripts/error_on_dirty.sh
+      - run: node ./scripts/validate_changelog.js
+      - run: git submodule sync && git submodule update --init
+      - restore_cache:
+          keys:
+            - ccache-debug-linux-x86_64-{{ .Branch }}
+            - ccache-debug-linux-x86_64
+      - run: npm install --ignore-scripts
+      - run: mkdir build
+      - run: ls -la /usr/bin/
+      - run: clang-5.0 --version
```

Comment:
Is this needed in this workflow? I guess we're building with GCC here.

Reply:
No, removed